include/deal.II/lac/scalapack.templates.h

```diff
@@ -51,8 +59,7 @@ extern "C"
    *
    * https://www.ibm.com/support/knowledgecenter/en/SSNR5K_4.2.0/com.ibm.cluster.pessl.v4r2.pssl100.doc/am6gr_dbpnf.htm
    */
-  void
-  Cblacs_pinfo(int *rank, int *nprocs);
+  void Cblacs_pinfo(int *rank, int *nprocs);
```

Comment:
I'm curious why this got changed here. I don't blame this patch in particular, but I'd like to understand the reasons for the change at this spot and similar ones further down and in #18493.

Reply:
I have long given up trying to understand what clang-format does :-) I suspect that it gets confused about the meaning of `DEAL_II_NAMESPACE_OPEN`, but why the presence or absence of a comment makes a difference, I really have no idea about.